src/test/java/mate/academy/bookstore/service/impl/CartItemServiceImplTest.java

```diff
@@ -28,62 +37,45 @@ class CartItemServiceImplTest {
     @Test
     @DisplayName("Verify update() method works for CartItemServiceImpl")
     public void updateCartItem_ValidRequestDto_ReturnCartItemDto() {
-        //Given
-        CartItem cartItem = new CartItem()
-                .setQuantity(10)
-                .setId(1L);
+        CartItem cartItem = validCartItem();
         UpdateQuantityRequestDto requestDto = new UpdateQuantityRequestDto(3);
-        CartItemDto cartItemDto = new CartItemDto(
-                cartItem.getId(),
-                null,
-                null,
-                cartItem.getQuantity());
+        CartItemDto cartItemDto = validUpdateCartItemDto();
 
-        Mockito.when(cartItemRepository.findById(Mockito.anyLong()))
+        when(cartItemRepository.findById(anyLong()))
                 .thenReturn(Optional.of(cartItem));
-        Mockito.when(cartItemRepository.save(cartItem)).thenReturn(cartItem);
-        Mockito.when(cartItemMapper.toDto(cartItem)).thenReturn(cartItemDto);
+        when(cartItemRepository.save(cartItem)).thenReturn(cartItem);
+        when(cartItemMapper.toDto(cartItem)).thenReturn(cartItemDto);
 
-        //When
-        CartItemDto updatedCartItemDto = cartItemService.update(requestDto, Mockito.anyLong());
+        CartItemDto updatedCartItemDto = cartItemService.update(requestDto, anyLong());
 
-        //Then
-        Assertions.assertEquals(cartItemDto, updatedCartItemDto);
-        Mockito.verify(cartItemMapper, Mockito.times(1)).toDto(Mockito.any());
-        Mockito.verify(cartItemRepository, Mockito.times(1)).save(Mockito.any());
-        Mockito.verify(cartItemRepository, Mockito.times(1)).findById(Mockito.any());
-        Mockito.verifyNoMoreInteractions(cartItemMapper, cartItemRepository);
+        assertEquals(cartItemDto, updatedCartItemDto);
+        verify(cartItemMapper).toDto(any());
+        verify(cartItemRepository).save(any());
```

Comment:
it's redundant to verify that stubbed method was called (you can check it in the method's documentation)